src/bssl.rs

```diff
@@ -14,10 +14,25 @@
 
 use {c, error};
 
-pub fn map_result(bssl_result: c::int) -> Result<(), error::Unspecified> {
-    match bssl_result {
-        1 => Ok(()),
-        _ => Err(error::Unspecified),
+/// A `c::int` returned from a foreign function containing **1** if the function was successful
+/// or **0** if an error occurred. This is the convention used by BoringSSL.
+#[must_use]
+#[repr(transparent)]
+#[derive(Debug, Clone, Copy)]
```

Comment:
Let's put `#[derive(...)]` first unless there's some reason not to. I usually sort these alphabetically when I remember to.